docs/settings/triggers/porcupine.md

```diff
@@ -0,0 +1,54 @@
+## Parameters
+
+| parameter    | required | type    | default | choices         | comment                                                                                          |
+|--------------|----------|---------|---------|-----------------|--------------------------------------------------------------------------------------------------|
+| keyword      | TRUE     | string  |         |                 | Path to the porcupine wake word. The path can be absolute or relative to the brain file          |
+| sensitivity  | FALSE    | string  | 0.5     | between 0 and 1 | Increasing the sensitivity value lead to better detection rate, but also higher false alarm rate |
+| input_device | FALSE    | integer | default | 				| Select the input device, otherwise the default device will be used        		               |
+| tiny_keyword | FALSE 	  | string	| false   | true/false      | If true you can use tiny keywords, these accuracy is slightly lower than the standard model but it consumes considerably less resources |
+
+## Example settings
+
+This is the trigger engine that will catch your magic work to wake up Kalliope. With porcupine we need different keywords for different platforms. The example use the keyword "porcupine" for the raspberry.
+
+```yaml
+default_trigger: "porcupine"
+
+# Trigger engine configuration
+triggers:
+  - porcupine:
+      keyword: "trigger/porcupine/porcupine_raspberrypi.ppn"
+```
+To use multiple keywords with different sensitivities
+```yaml
+default_trigger: "porcupine"
+
+# Trigger engine configuration
+triggers:
+  - porcupine:
+      keyword:
```

Comment:
Just reading the doc and not the code, but would it be better to have a definition like this:
```
triggers:
  - porcupine:
      keyword:
        - porcupine:
            ppn: "trigger/porcupine/porcupine_raspberrypi.ppn"
            sensitivity: 0.4
          - blueberry_raspberrypi:
            ppn: "trigger/porcupine/blueberry_raspberrypi.ppn"
            sensitivity: 0.4
```

Reply:
weird indents in what you are showing @bacardi55

Reply:
lol sorry on my phone.
The idea is simple though, group both option (ppn file + sensitivity) in the same definition. otherwise it seems weird to define ppn in one place and sensitivity in another and force the user to keep the same orders for both^^

Reply:
yes you are right. If the trigger can process multiple hotword in the same time, your syntax is easier to read.

Reply:
Yes you are right, it would be much easier to handle the settings. I will take care of it, thanks for the suggestion!